Foosball/FoosballWebsite/Controllers/HomeController.cs

```diff
@@ -0,0 +1,14 @@
+using Microsoft.AspNetCore.Mvc;
+
+namespace LiveGameFeed.Controllers
+{
+    public class HomeController : Controller
+    {
+        public HomeController() { }
```

Comment:
Default constructor is not needed to write. It is automatically added then compiling project